tools/Google.Cloud.Tools.ReleaseTool/GitHub.cs

```diff
@@ -0,0 +1,109 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License").
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at 
+//
+// https://www.apache.org/licenses/LICENSE-2.0 
+//
+// Unless required by applicable law or agreed to in writing, software 
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and 
+// limitations under the License.
+
+using Octokit;
+
+namespace Google.Cloud.Tools.ReleaseTool;
```

Comment:
Meant to say before... given that this doesn't do *all* (or even most) of what release-tool does, I'd just call this ReleaseReporter or maybe ReleaseProgressReporter.